skore-ui/src/assets/styles/_variables.css

```diff
@@ -1,102 +1,95 @@
 /*
-  We try to follow a naming structure of our variables.
-
-  `--component-property-variant-kind-modifier`
-
-  - component: the domain where this variable is applied (text color, transitions, ...)
-  - property: the CSS properties related to this variable
-  - variant: where this apply, is it an error, a success, an elevation, ... ?
-  - kind: is it small, large, high, higher, ... ?
-  - modifier: is it hover, visited, ... ?
+ Those variables come from the design system in figma.
 */
-
 :root {
-  /* typography */
-  --text-color-title: var(--color-primary);
-  --text-size-small: 12px;
-  --text-size-normal: 14px;
-  --text-size-highlight: 14px;
-  --text-size-title: 14px;
-  --text-weight-normal: 300;
-  --text-weight-highlight: 400;
+  --root-font-size: 14px;
 
-  /* buttons */
-  --button-background: var(--color-primary);
+  /* typography */
+  --font-size-xs: 0.85rem;
+  --font-size-sm: 1rem;
+  --font-size-md: 1.14rem;
+  --font-size-lg: 1.42rem;
+  --letter-spacing: -0.015em;
+  --font-weight-regular: 400;
+  --font-weight-medium: 500;
 
   /* spacings */
-  --spacing-padding-large: 24px;
-  --spacing-padding-normal: 16px;
-  --spacing-padding-small: 8px;
-  --spacing-gap-small: 5px;
-  --spacing-gap-normal: 10px;
-  --spacing-gap-large: 20px;
-
-  /* borders */
-  --border-width-normal: 1px;
-  --border-width-small: 0.34px;
-  --border-radius: 4px;
-
-  /* misc */
-  --transition-duration: 0.3s;
-  --transition-easing: ease;
-
-  /* brand colors */
-  --color-blue: #2329ff;
-  --color-orange: #f08b30;
+  --spacing-4: 0.28rem;
```

Comment:
using rem is nice! do we even need to set `root-font-size`?

Reply:
Yes we need to set the root size to 14px as designed. 
You can see that its done in `_typograpy.css`.